RELEASENOTES.md

```diff
@@ -1,5 +1,6 @@
 # Release Notes
 ### 3.0.0 (unreleased)
+* The sms_summary_page not and sms_ask option can now be used together.
```

Comment:
The description here doesn't seem to make sense.  Can you move bug fixes to the bottom of the list?

Reply:
Hmm weird yes that is confusing will do

Reply:
I forgot to include the ticket number in each enhancement / bug fix.  You should follow the pattern established in prior release notes.